Update README.md (#253)
fly-apps/terraform-provider-fly: README.md

# fly.io terraform provider

[![Tests](https://github.com/fly-apps/terraform-provider-fly/actions/workflows/test.yml/badge.svg)](https://github.com/fly-apps/terraform-provider-fly/actions/workflows/test.yml)

> **This is not an officially supported or maintained project and is not a recommended method of deployment to Fly.io.**

Author: ```suggestion
**This project is not currently maintained, and is not a recommended method of deployment to Fly.io.**
```
